tritonuas/obcpp
# Feat/conn status

Related PR with https://github.com/tritonuas/gcs/pull/121

Information for both here

On the OBC it implements a /connection route and a / route for connection testing purposes for the GCS. It also resets the container to :main because :x86 is currently broken.

On the GCS side of things it implements the obc connection page. There is shit CSS but it displays the info correctly.

Also, it links up the main status indicators on the topbar, but the only one that actually provides meaningful info is the obc one as we aren't really testing connection on the others I think

## How To Test / Expected Behavior

Spawn up this branch's obcpp in one terminal.

Pull newest GCS and run `make build-docker` and `make run-compose`

Open `localhost:5000` in web browser

verify that after 2s the OBC connection is green

then visit the OBC connection page and see if it is giving the information (should be all false/null values because nothing is really connected)

## Notes

General code quality on the GCS side is very bad. Also the CSS is awful. But It should work.
